config/betterquesting/DefaultQuests/QuestLines/EndgameGoals-AAAAAAAAAAAAAAAAAAAAHQ==.json

```diff
@@ -347,7 +347,7 @@
     "43:10": {
       "questIDHigh:4": 0,
       "sizeX:3": 24,
-      "x:3": 252,
+      "x:3": 240,
```

Comment:
the quests are assymmetric now. don't think that's better

Reply:
please provide a screenshot of what quest is that

Reply:
hm I assume you mean thermosink radiator quest, I can revert it but the lines will overlap, is that ok?

Reply:
yea